src/sidebar/store/use-store.js

```diff
@@ -35,18 +29,26 @@ class CacheEntry {
 }
 
 /**
- * Return a wrapper around the `store` service that UI components can use to
- * extract data from the store and call actions on it.
+ * Return a wrapper around a store that UI components can use to read from and
+ * modify data in it.
  *
- * Unlike using the `store` service directly, the wrapper tracks what data from
- * the store the current component uses, via selector methods, and re-renders the
- * component when that data changes.
+ * Unlike using the store directly, the wrapper tracks what data from
+ * the store the current component uses, by recording calls to selector methods,
+ * and re-renders the components when the results of those calls change.
  *
  * The returned wrapper has the same API as the store itself.
  *
  * @example
+ *   // A hook which encapsulates looking up the specific store instance,
+ *   // eg. via `useContext`.
+ *   function useAppStore() {
+ *     // Get the store from somewhere, eg. a prop or context.
+ *     const appStore = ...;
+ *     return useStore(store);
+ *   }
```

Comment:
Should this be

```suggestion
 *   function useAppStore() {
 *     // Get the store from somewhere, eg. a prop or context.
 *     const appStore = ...;
 *     return useStore(appStore);
 *   }
```

or perhaps

```suggestion
 *   function useAppStore() {
 *     // Get the store from somewhere, eg. a prop or context.
 *     const store = ...;
 *     return useStore(store);
 *   }
```

?